src/Microsoft.Extensions.Caching.Redis/project.json

```diff
@@ -11,10 +11,21 @@
   },
   "dependencies": {
     "Microsoft.Extensions.Caching.Abstractions": "1.0.0-*",
-    "Microsoft.Extensions.Options": "1.0.0-*",
-    "StackExchange.Redis.StrongName": "1.0.333"
+    "Microsoft.Extensions.Options": "1.0.0-*"
   },
   "frameworks": {
-    "net451": {}
+    "net451": {
+      "StackExchange.Redis.StrongName": "1.0.488"
+    },
+    "dnxcore50": {
+      "dependencies": {
+        "StackExchange.Redis.StrongName": "1.1.553-alpha"
```

Comment:
Hmm when will this ship an RTM version? We'd like to avoid depending on other pre-release packages because once ours go RTM, they can no longer depend on other pre-rel packages.